### DISCUSSION TITLE
Blob for Infra Gravity waves

### DISCUSSION THREAD

**Discussion by meenakshisreejith** [NOAA-EMC/WW3#565] [Q&A]
Hello All
Im running WAVEWATCH III model from infra gravity waves in the frequency width 0.06-0.3 Hz. The run is completed successfully but the output has blobas. I am running for a 0.5degree global grid. Can you please help in this regard?

**Comment by JessicaMeixner-NOAA** [comment]
@meenakshisreejith I'm not sure I understand what the issue is.  Can you provide some additional information?

**Reply by meenakshisreejith** [reply]
I am trying to explore infra gravity simulation using a global grid 0f 0.5 deg. Spectral resolution is 24 frequencies from 0.006  to 0.05 and 36 directions. Model results are unrealistic and blowing up. Can you please help regarding this?

**Reply by JessicaMeixner-NOAA** [reply]
What are your time steps that you are using?

**Reply by meenakshisreejith** [reply]
Hello Maam
These are my time steps:
1200.  600.  700.  30.

**Reply by JessicaMeixner-NOAA** [reply]
What is your highest latitude?

**Reply by meenakshisreejith** [reply]
This is my grid detail:

LON       LONGITUDE          720mr   0E                   0.5W
 LAT       LATITUDE           321 r   80S                  80N

**Reply by JessicaMeixner-NOAA** [reply]
The CFL condition can be found here: https://github.com/NOAA-EMC/WW3/wiki/FAQs-page#cfl-condition 
for your set-up: 
123766 * Δx * cosd(max(latitude))*f
123766*0.5*cosd(80)*0.006
64.4752
So you need much smaller time steps.
